fluentd-gcp-image/Dockerfile

```diff
@@ -45,6 +45,9 @@ RUN BUILD_DEPS="make gcc g++ libc6-dev ruby-dev libffi-dev" \
               /var/log/* \
               /var/tmp/*
 
+# For systems without journald.
+RUN mkdir -p /var/log/journal
```

Comment:
Oh, that actually won't work, because `/var/log` is mounted in runtime and this directory is dropped

Reply:
Yes I seem to have noticed the same thing :-)

I'll rework that and send a single squashed commit. Hang on.